### PR TITLE
update(retina): remove network observability from cli

### PR DESCRIPTION
**Pull request by snguyen64**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
This removes network observability from the preview cli. 

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/aks-preview/HISTORY.rst [hunk 1]
```diff
@@ -13,6 +13,7 @@ Pending
 +++++++
 * Update --enable-advanced-network-observability description to note additional costs and add missing flag to create command.
 * Add etag support (--if-match, --if-none-match) to some aks commands for optimistic concurrency control.
+* Remove --enable-network-observability and --disable-network-observability from aks create and update commands.
```
Comment (FumingZhang) [inline src/aks-preview/HISTORY.rst:16]: This is a breaking change as you removed several options.

Please set the version number as `5.0.0b1` (update `VERSION` in setup.py accordingly) and add `[BREAKING CHANGE]:` to the history note.

File: src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py [hunk 2]
```diff
@@ -12836,108 +12836,6 @@ def test_aks_update_upgrade_settings(self, resource_group, resource_group_locati
...
         )
 
-    @AllowLargeResponse()
-    @AKSCustomResourceGroupPreparer(
-        random_name_length=17,
-        name_prefix="clitest",
-        location="eastus",
-        preserve_default_location=True,
-    )
-    def test_aks_update_enable_network_observability(
```
Comment (FumingZhang) [inline src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py:12846]: Please remove the corresponding recording file, for example, for test case `test_aks_update_enable_network_observability`, the recording file is `src/aks-preview/test/latest/recordingstest_aks_update_enable_network_observability.yaml`

File: src/aks-preview/HISTORY.rst [hunk 3]
```diff
@@ -14,6 +14,10 @@ Pending
 * Update --enable-advanced-network-observability description to note additional costs and add missing flag to create command.
 * Add etag support (--if-match, --if-none-match) to some aks commands for optimistic concurrency control.
```
Comment (FumingZhang) [inline src/aks-preview/HISTORY.rst:15]: Please include these notes in 5.0.0b1

File: src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py [hunk 4]
```diff
@@ -12836,108 +12836,6 @@ def test_aks_update_upgrade_settings(self, resource_group, resource_group_locati
             checks=[self.is_empty()],
         )
 
-    @AllowLargeResponse()
```
Comment (FumingZhang) [inline src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py:12839]: According to the error msg, some unit test cases were not deleted

> src/aks-preview/azext_aks_preview/tests/latest/test_managed_cluster_decorator.py:955: AttributeError
> - generated xml file: /home/cloudtest/.azdev/env_config/mnt/vss/_work/1/s/env/test_results.xml -
> =========================== short test summary info ============================
> FAILED src/aks-preview/azext_aks_preview/tests/latest/test_managed_cluster_decorator.py::AKSPreviewManagedClusterContextTestCase::test_mc_get_enable_network_observability

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**